lib/crypto.js

```diff
@@ -515,7 +515,7 @@ DiffieHellman.prototype.setPrivateKey = function(key, encoding) {
 
 function ECDH(curve) {
   if (typeof curve !== 'string')
-    throw new TypeError('curve should be a string');
+    throw new TypeError('"curve" argument should be a string');
```

Comment:
We don't have a compromise on this yet (should vs must) see https://tools.ietf.org/html/rfc2119